core/common/src/main/java/alluxio/wire/FileInfo.java

```diff
@@ -61,6 +61,7 @@
   private String mUfsFingerprint = Constants.INVALID_UFS_FINGERPRINT;
   // A list of String ACL entries, to support json serialization.
   private ArrayList<String> mAclEntries = new ArrayList<>();
+  private ArrayList<String> mDefaultAclEntries = new ArrayList<>();
```

Comment:
please update `equals`, `hashCode`, and `toString` to include this.

Reply:
done